lib/platform/require.js

```diff
@@ -1,12 +1,12 @@
~import Process from './process';
~import Socket from './socket';
~
~/**
~ * Initialize a new `Module`.
~ * @param {string} id The module identifier
~ * @public
~ */
~export default function Module(id) {
```

Comment:
<!--
  0 failure: 
  1 warning:  lib/platform/requ...
  
  
  DangerID: danger-id-Danger;
  File: lib/platform/require.js;
  Line: 1;
-->

- :warning: lib/platform/require.js line 1 – Import and export declarations are not supported yet. (node/no-unsupported-features/es-syntax)